### PR TITLE
Update LSP6 relay call guide

### PR DESCRIPTION
**Pull request by fhildeb**
Todo:

- [x] Test script in playground
- [x] Grammar checks

### CODE REVIEW

File: docs/learn/expert-guides/key-manager/execute-relay-transactions.md [hunk 1]
```diff
@@ -8,91 +8,71 @@ import TabItem from '@theme/TabItem';
...
-  
-  <TabItem value="web3js" label="web3.js">
+:::info Execution Rights
 
-```typescript title="Imports & Constants"
-import UniversalProfileContract from '@lukso/lsp-smart-contracts/artifacts/UniversalProfile.json';
-import KeyManagerContract from '@lukso/lsp-smart-contracts/artifacts/LSP6KeyManager.json';
-import { EIP191Signer } from '@lukso/eip191-signer.js';
-import Web3 from 'web3';
+Even when relaying the transaction using a third-party, you will need execute permission is necessary. Otherwise, unauthorized parties would be able to send transactions to your relayer address.
```
Comment (CJ42) [inline docs/learn/expert-guides/key-manager/execute-relay-transactions.md:60]: Suggest adding link to the permission in the LSP6 standard page. And remove the 2nd sentence which contradicts the first.

```suggestion
To execute relay transactions using a third-party add, you will need the `EXECUTE_RELAY_CALL` permission.
```

**Comment by Hugoo** [comment]
@fhildeb what is the status on that PR?

**Comment by fhildeb** [comment]
Ready to be merged- tested and applied grammar checks & suggestions from Jean.

Related playground PR: https://github.com/lukso-network/lukso-playground/pull/52

File: docs/learn/expert-guides/key-manager/execute-relay-transactions.md [hunk 2]
```diff
@@ -8,93 +8,73 @@ import TabItem from '@theme/TabItem';
...
+To execute relay transactions using a third-party add, you will need the [`EXECUTE_RELAY_CALL`](https://docs.lukso.tech/standards/universal-profile/lsp6-key-manager/#permissions) permission.
 
-// This is the version relative to the LSP25 standard, defined as the number 25.
-import { LSP25_VERSION } from '@lukso/lsp-smart-contracts/constants';
+:::
 
-const web3 = new Web3('https://rpc.testnet.lukso.network');
-const universalProfileAddress = '0x...';
-const recipientAddress = '0x...';
+After installation, we can import the contract artifacts and set up relevant constants. To encode the transaction, you need the address of the Universal Profile smart contract and the private key of a controller key with sufficient [LSP6 Permissions](../../../standards/universal-profile/lsp6-key-manager.md#permissions) to **execute** the transaction. The controller, addresses, and artifacts will then be used to set up the contract instances for the [Universal Profile](../../../standards/universal-profile/lsp0-erc725account.md) and [Key Manager](../../../standards/universal-profile/lsp6-key-manager.md) you will interact with.
```
Comment (CJ42) [inline docs/learn/expert-guides/key-manager/execute-relay-transactions.md:64]: Suggest rewording back to how it was, and maybe add 1 extra sentence:

```suggestion
To encode a transaction, we need the address of the Universal Profile smart contract and the private key of a controller key with sufficient [LSP6 permissions](../../../standards/universal-profile/lsp6-key-manager.md#permissions) to execute the transaction.
First, create an instance of the Universal Profile contract and its Key Manager.
```

File: docs/learn/expert-guides/key-manager/execute-relay-transactions.md [hunk 3]
```diff
@@ -103,107 +83,146 @@ const provider = new ethers.JsonRpcProvider(
...
 
-<!-- prettier-ignore-end -->
-
   </TabItem>
 
-  <TabItem value="ethersjs" label="ethers.js">
+</Tabs>
 
-<!-- prettier-ignore-start -->
+:::caution Caution when using a Private Key
```
Comment (CJ42) [inline docs/learn/expert-guides/key-manager/execute-relay-transactions.md:152]: This is a good callout 👌 . Would even put that in danger.

```suggestion
:::danger Caution when using your controller's Private Key
```

File: docs/learn/expert-guides/key-manager/execute-relay-transactions.md [hunk 4]
```diff
@@ -212,358 +231,291 @@ const abiPayload = universalProfile.methods
...
+    //                 fecafecafecafe00000000000000000000000000000000000000000000000029a2 
+    //                 241af62c0000000000000000000000000000000000000000000000000000000000
+    //                 000000008000000000000000000000000000000000000000000000000000000000
+    //                 00000000`
+    abiPayload,         
   ],
 );
 
-let eip191Signer = new EIP191Signer();
+// Instanciate EIP191 Signer
```
Comment (CJ42) [inline docs/learn/expert-guides/key-manager/execute-relay-transactions.md:305]:
```suggestion
// Instantiate EIP191 Signer
```

File: docs/learn/expert-guides/key-manager/execute-relay-transactions.md [hunk 5]
```diff
@@ -212,358 +231,291 @@ const abiPayload = universalProfile.methods
...
+const eip191Signer = new EIP191Signer();
 
-let { signature } = await eip191Signer.signDataWithIntendedValidator(
+/**
```
Comment (CJ42) [inline docs/learn/expert-guides/key-manager/execute-relay-transactions.md:308]: Suggest removing this code comment block for brevity. The code and function parameters are self explanatory themselves.

File: docs/learn/expert-guides/key-manager/execute-relay-transactions.md [hunk 6]
```diff
@@ -103,107 +83,146 @@ const provider = new ethers.JsonRpcProvider(
...
+:::
 
-<!-- prettier-ignore-end -->
+## Prepare the Relay Call
 
-  </TabItem>
+After setting up both the [Universal Profile](../../../standards/universal-profile/introduction.md) and [Key Manager](../../../standards/universal-profile/lsp6-key-manager.md) of the signing person, we can continue to prepare all the relay call parameters. These are crucial for ensuring secure and authenticated transactions. Here is what you will need:
 
-</Tabs>
+- **`nonce` of the controller**: This value can be retrieved by calling the [`getNonce`](../../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#getnonce) function on the [Key Manager](../../../standards/universal-profile/lsp6-key-manager.md) associated with the profile. The `nonce` of an EOA is incremented with each transaction, ensuring their uniqueness.
```
Comment (CJ42) [inline docs/learn/expert-guides/key-manager/execute-relay-transactions.md:162]: The nonce can be from an EOA or a contract. Suggest removing this

```suggestion
- **`nonce` of the controller**: can be retrieved via the [`getNonce`](../../../contracts/contracts/LSP6KeyManager/LSP6KeyManager.md#getnonce) function on the [Key Manager](../../../standards/universal-profile/lsp6-key-manager.md) associated with the Universal Profile.
```

**Review by CJ42 — CHANGES_REQUESTED** [review]

**Comment by fhildeb** [comment]
Thanks for the review, good input. 🙏🏻 Applied suggestions and grammar on code snippets.

> Regarding client-side/server-side differentiation: I initially wanted to point out that executing a relay call involves two steps: preparation on the client side and deployment on the server/relay service. However, I agree that the bullet point list caused more confusion—a clear sentence fits much better. We don't have to state that relay services are server-side. ✅

**Review by CJ42 — APPROVED** [review]
